### DISCUSSION TITLE
Filters - Configuration | Privileges

### DISCUSSION THREAD

**Discussion by s2b-git** [YetiForceCompany/YetiForceCRM#14871] [How to]
Gents what is behind column "Privileges" in Filters - Configuration?
Cannot figure out how to use this radio

Best

**Comment by rskrzypczak** [comment]
It just determines whether the filter can be edited.
Take a look at this panel in the developer version. 
From version 6.2, it will be possible to better manage filter permissions.
![image](https://user-images.githubusercontent.com/10221999/123513028-0ce40380-d68b-11eb-885a-4d79897e4868.png)

**Comment by s2b-git** [comment]
Thank you
